screencastr.thor

```diff
@@ -69,4 +79,16 @@ class Screencastr < Thor
     -r #{options[:framerate] || 30} \
     -map '[vout]' -map '[aout]' #{out_file}`
   end
+
+  desc "brand IN_FILE OUT_FILE", "Transcode, watermark, and add bumpers, all in one command"
+  def brand(in_file, out_file)
+    ProcessHelpers.resolution_require_value(options[:resolution])
+    ext = File.extname(out_file)[1..-1]
+    invoke :add_watermark, [in_file, "watermarked.#{ext}"]
+    begin
+      invoke :add_bumpers, ["watermarked.#{ext}", out_file]
+    ensure
+      File.delete("watermarked.#{ext}") if File.exists?("watermarked.#{ext}")
+    end
+  end
```

Comment:
This task does all the branding in one go: adds a watermark to the screencast, and then adds the bumpers. Here, I am using `invoke`, since I'm not calling the same task more than once.

Reply:
This looks good, no way to avoid the intermediate file here as far as I can tell.